README.md

```diff
@@ -18,6 +18,11 @@ To run the server, execute:
 
 The service is currently configured to start on port 6001 as we allow access to localhost 6001-6010 from our build slave side car containers.
 
+## Service configuration
+When running the service locally you can update [config.js](config.js), or set the following environment variables:
+- **port**: defaults the environment variable named **APP_PORT** if set.  Otherwise, it's set to 6001
+- **captureAllPages**: defaults to the environment variable named **CAPTURE_ALL_PAGES** if set.  Otherwise, it's set to **false**.  When set to false the service will capture a single version of each page by path.  Setting this to **true** will capture every page uploaded to the service.
+- **pagesParentPath**: defaults to the value of the **WORKSPACE** environment variable (for use in Jenkins).  If not set, pages are saved to a sub-directory of the current working directory 
```

Comment:
One thing I noticed regarding the pagesParentPath variable is, almost every one of us would have a $WORKSPACE variable configured locally as this is being used by Service Manager. So what happens is the pages are by default captured in the local $WORKSPACE directory and not in `__dirname`.